lib/mobility/active_record.rb

```diff
@@ -11,13 +11,7 @@ module ActiveRecord
     require "mobility/active_record/uniqueness_validator"
 
     def self.included(model_class)
-      model_class.class_eval do
-        unless const_defined?(:UniquenessValidator)
```

Comment:
Figured out the problem: this should be `const_defined?(:UniquenessValidator, false)`.